collectors/gsuite/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "gsuite-collector",
-    "version": "1.0.1",
+    "version": "1.0.2",
```

Comment:
gsuite is version 1.1.6 in our pipeline. could you please bring this up to speed

Reply:
👍